### PR TITLE
Bb]itcoin grammar change

### PR DESCRIPTION
**Pull request by chanan-sack**
all mentiones of [Bb]itcoin have been changed accordingly to requested set layout

### CODE REVIEW

File: code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "\n",
     "    $ pip3 install ethereum\n",
     "    \n",
-    "    $ pip3 install bitcoin\n",
+    "    $ pip3 install Bitcoin\n",
```
Comment (aantonop) [inline code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb]: this one is bitcoin, small b. Package names are case-sensitive

File: code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb [hunk 2]
```diff
@@ -33,8 +33,8 @@
     "\n",
     "# Vitalik Buterin's Python Library for Bitcoin\n",
     "# No longer maintained!\n",
-    "# https://pypi.python.org/pypi/bitcoin/1.1.42\n",
-    "import bitcoin\n",
+    "# https://pypi.python.org/pypi/Bitcoin/1.1.42\n",
```
Comment (aantonop) [inline code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb]: small b

File: code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb [hunk 3]
```diff
@@ -33,8 +33,8 @@
...
+    "import Bitcoin\n",
```
Comment (aantonop) [inline code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb]: small b

File: what-is.asciidoc [hunk 4]
```diff
@@ -26,7 +26,7 @@ The components of an open, public, blockchain are (usually):
 * A consensus algorithm (e.g. Proof-of-Work) that decentralizes control over the blockchain, by forcing participants to compete and constraining them by the consensus rules.
 * One or more open-source software implementations of the above.
 
-All or most of these components are usually combined in a single software client. For example, in bitcoin, the reference implementation is developed by the _Bitcoin Core_ open source project, implemented as the +bitcoind+ client. In Ethereum, rather than a reference implementation, there is a _reference specification_, a mathematical description of the system in the <<yellowpaper>>. There are a number of clients, which are built according to the reference specification.
+All or most of these components are usually combined in a single software client. For example, in Bitcoin, the reference implementation is developed by the _Bitcoin Core_ open source project, implemented as the +Bitcoind+ client. In Ethereum, rather than a reference implementation, there is a _reference specification_, a mathematical description of the system in the <<yellowpaper>>. There are a number of clients, which are built according to the reference specification.
```
Comment (aantonop) [inline what-is.asciidoc]: Bitcoind is small b, it's case sensitive command, not a proper noun

**Review by aantonop — CHANGES_REQUESTED** [review]

File: code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb (hunk 1 above)
Comment (fjrojasgarcia) [reply code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb]: Yes, be careful with this kind of mega refactoring.

File: code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb (hunk 2 above)
Comment (fjrojasgarcia) [reply code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb]: A jupyter notebook is pure code, please be so careful with code changes, and please retest it with the proposed change.

File: what-is.asciidoc (hunk 4 above)
Comment (fjrojasgarcia) [reply what-is.asciidoc]: Same here, as a general pattern, I'd recommend avoid book code refactorization.

It is vital to put yourself in the place of the reader who tries to learn with the code offered as an example in the book. This should always work and faithfully reproduce the operations proposed in the content of the book.

**Comment by chanan-sack** [comment]
understood, was being made on mobile device and i did not realize i was making changes to case sensitive code

**Comment by chanan-sack** [comment]
fixed issues and merge conflict, should be final fix

**Pull request merged**